doc/dev/development.md

```diff
@@ -140,11 +149,18 @@ git remote set-url --push upstream no_push
 
 ### Committing changes to your fork
 
-When you're happy with some changes, you can commit them and push them to your fork:
+When you're happy with some changes, you can commit them to your fork:
```

Comment:
The original wording is more precise.

You commit to your local repo, and then you push into your fork. Finally, the content will eventually be pulled from your fork into the upstream branch.